chirp/drivers/tk760g.py

```diff
@@ -813,7 +813,7 @@ def _prep_data(self):
         # building the data for the memmap
         fdata = ""
 
-        for k, v in data.iteritems():
+        for k, v in list(data.items()):
```

Comment:
This is a nit, but I think just `data.items()` is fine here, no need to listify it first.

Reply:
Done!!!